libs/designsystem/form-field/src/form-field.component.ts

```diff
@@ -159,6 +166,17 @@ export class FormFieldComponent
       this.isRegistered = true;
       this.dispatchLoadEvent();
     }
+
+    // TODO: Is ngAfterContentChecked the right lifecycle method to hook into?
+    // Decide if default calendar icon for date input should be shown
+    this.showDefaultIcon = this.shouldShowDefaultIcon();
```

Comment:
I think we should add calendar or datepicker or something similar here for clarity, as we are in formfield that supports multiple types of inputs

```suggestion
    this.showDefaultCalendarIcon = this.shouldShowDefaultCalendarIcon();
```